components/TableContents.js

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+import Links from './Links'
+
+export default function TableContents(props) {
+  const lang = props.lang?.trim?.() === 'fr' ? 'fr' : 'en'
+  const { id, sectionList } = props
+  return (
+    <>
+      <section id={id}>
+        <h2 className="font-display text-2xl leading-[26px] font-bold text-gray-darker">
+          {lang === 'en' ? 'On this page' : 'Sur cette page'}
+        </h2>
+        <ul className="pl-[22px] ml-3.5">
+          {sectionList.map((option, index) => {
+            return (
+              <li
+                key={index}
+                className={
+                  'underline underline-offset-4 text-20px text-gray-darker underline-offset-3.1 decoration-1 pr-1 list-outside list-disc'
+                }
+              >
+                <Links
+                  href={option.link}
+                  id={`tableLink${index + 1}`}
+                  text={option.name}
+                  linkStyle="cardActionLink"
+                />
```

Comment:
We aren't using a large majority of this file so I would just do away with it since we can just sub in the most important stuff like so:

```suggestion
                <Link
                  className="text-deep-blue-dark font-body text-xl hover:text-blue-hover focus:text-blue-hover"
                  href={option.link}
                  id={`tableLink${index + 1}`}
                  data-cy={`tableLink${index + 1}`}
                  aria-label={option.name}
                >
                {option.name}
                </Link>
```
I don't see the need to keep this component in since I moved the ternary directly into the Language component. The only reason to keep this would be for easy access to styling but I don't think that's a good enough reason to do so.

Reply:
moved link to local file instead.